packages/xo-web/src/xo-app/vm-import/index.js

```diff
@@ -264,6 +264,10 @@ export default class Import extends Component {
   }
 
   _handleDrop = async files => {
+    this.setState({
+      vms: [],
+    })
+
```

Comment:
Why this change?

It seems unrelated to the XML fixes…

Reply:
it resets the display when we import a new file.